modules/openy_features/openy_node/modules/openy_node_blog/openy_node_blog.module

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * @file
+ * Main file for the OpenY Node Blog module.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\State\StateInterface;
+
+/**
+ * Implements hook_form_alter().
+ */
+function openy_node_form_alter(&$form, FormStateInterface $form_state, $form_id) {
```

Comment:
I recommend to use hook_form_FORM_ID_alter(), see documentation https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Form%21form.api.php/function/hook_form_FORM_ID_alter/8.3.x